docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
```

Comment:
This includes docs for the `?master_timeout` parameter, but the cluster stats API doesn't talk to the master so this parameter is not relevant.

Reply:
I replaced this with only the `timeout` parm with e1b6d2c8cc2.